frontend/src/components/Worksheet/WorksheetMobileCalendarListItem.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Row, Col, ListGroup } from 'react-bootstrap';
+import styled, { withTheme, type DefaultTheme } from 'styled-components';
+import styles from './WorksheetCalendarListItem.module.css';
+import WorksheetToggleButton from './WorksheetToggleButton';
+import WorksheetHideButton from './WorksheetHideButton';
+import { useWorksheet } from '../../contexts/worksheetContext';
+import { Listing } from '../Providers/FerryProvider';
+
+// Listgroup Item for worksheet list item
+const StyledListItem = styled(ListGroup.Item)`
+  background-color: transparent;
+  border-color: ${({ theme }) => theme.border};
+  overflow: hidden;
+  transition:
+    border-color ${({ theme }) => theme.trans_dur},
+    background-color ${({ theme }) => theme.trans_dur},
+    color ${({ theme }) => theme.trans_dur};
+  &:hover {
+    cursor: pointer;
+    background-color: ${({ theme }) => theme.select_hover};
+  }
+  /* Hides icon until you hover over the list item */
+  .hidden {
+    opacity: 0;
+    transition: 0.05s opacity;
+  }
+  &:hover .hidden {
+    opacity: 1;
+  }
+  @media only screen and (max-width: 480px) {
+    .hidden {
+      opacity: 1;
+    }
+`;
```

Comment:
```suggestion
    }
  }
`;
```

...right? I don't know for sure